src/main/webapp/resources/js/pages/projects/share/ShareProject.jsx

```diff
@@ -37,15 +48,30 @@ export function ShareProject({ currentProject }) {
     );
   }, [projects]);
 
+  const handleSearch = (value) => {
+    const lowerValue = value.toLowerCase();
+    const available = projects.filter((project) =>
+      project.name.toLowerCase().includes(lowerValue)
+    );
+    console.log(available);
```

Comment:
Remove logging statement here and below

Reply:
Removed in aed74ad9ab207af386ac82efda7880ce51dc3fd7